build_tools/waf_dynamo.py

```diff
@@ -872,8 +868,7 @@ def _strip_executable(bld, platform, target_arch, path):
     if 'android' in platform:
         HOME = os.environ['USERPROFILE' if sys.platform == 'win32' else 'HOME']
         ANDROID_HOST = 'linux' if sys.platform == 'linux' else 'darwin'
-        build_tool = getAndroidBuildtoolName(target_arch)
-        strip = "%s/toolchains/%s-%s/prebuilt/%s-x86_64/bin/%s-strip" % (ANDROID_NDK_ROOT, build_tool, ANDROID_GCC_VERSION, ANDROID_HOST, build_tool)
+        strip = "%s/toolchains/llvm/prebuilt/%s-x86_64/bin/llvm-strip" % (ANDROID_NDK_ROOT, ANDROID_HOST)
```

Comment:
Does the 64 bit `strip`now`work on `armv7` executables?

Reply:
I can't find docs for it (it's always isn't so easy) but I tested it and it works